Rocket.Chat/API/Requests/General/PushTokenDeleteRequest.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PushTokenDeleteRequest.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 12/8/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import SwiftyJSON
+
+typealias PushTokenDeleteResult = APIResult<PushTokenDeleteRequest>
+
+class PushTokenDeleteRequest: APIRequest {
+    let requiredVersion = Version(0, 60, 0)
+
+    let method = "DELETE"
+    let path = "/api/v1/push.token"
```

Comment:
Maybe constants here would be interesting too...

Reply:
Some will be variables like rooms.upload/rid, so I'll think of a better solution later 🤔